plugins/storage/volume/linstor/src/main/java/com/cloud/hypervisor/kvm/storage/LinstorStorageAdaptor.java

```diff
@@ -197,6 +197,10 @@ public boolean deleteStoragePool(KVMStoragePool pool) {
         return deleteStoragePool(pool.getUuid());
     }
 
+    /**
+     * createPhysicalDisk will check if the resource wasn't yet created and do so, also it will make sure
+     * it is accessible from this node (MakeAvailable).
+     */
```

Comment:
should this javadoc be on the interface?

Reply:
I don't think so, as I guess other `createPhysicalDisk()` calls might just create the actual disk, but not check before if it is necessary.
Just wanted that to be documented somewhere, maybe I also got your question wrong and I should move the comment in the method?

Reply:
by no means, if this is specialised behaviour for the Linstor plugin this javadoc is in the right place!

Reply:
it just seemed something that all storage plugins are supposed to de ;)